tests/backend/invite/test_base.py

```diff
@@ -337,6 +337,14 @@ async def test_delete(
             pass
 
 
+@pytest.mark.trio
+async def test_user_invitation_already_member(alice, bob, backend, alice_backend_sock):
```

Comment:
would be good to add another similar test (or to parametrize this one ^^) that create another revoked user with similar human handle to make sure the invitation is allowed in such case